pennylane/qchem/openfermion_obs.py

```diff
@@ -978,4 +1020,66 @@ def molecular_hamiltonian(
 
     h_pl = qml.qchem.convert.import_operator(h_of, wires=wires, tol=convert_tol)
 
-    return h_pl, qubits
+    return h_pl, len(h_pl.wires)
+
+
+def _pyscf_integrals(
+    symbols,
+    coordinates,
+    charge=0,
+    mult=1,
+    basis="sto-3g",
+    active_electrons=None,
+    active_orbitals=None,
+):
+    r"""Compute pyscf integrals."""
```

Comment:
It might be worthwhile to document or have code comments on the steps in the integral computation.